examples/private-terraform-module/README.md

```diff
@@ -38,11 +38,13 @@ jobs:
         # IMPORTANT: add any required secrets to setup cloud credentials so Terraform can run
         # env:
 
-      - task: InfracostComment@0
-        displayName: Post the comment
-        inputs:
-          githubToken: $(githubToken)
-          path: /tmp/infracost.json
-          behavior: update # Create a single comment and update it. See https://github.com/infracost/infracost-azure-devops#infracostcomment for other options
+      - bash: |
+          infracost comment github \
```

Comment:
```suggestion
          # Create a single comment and update it. See https://www.infracost.io/docs/features/cli_commands/#comment-on-pull-requests for other options
          infracost comment github \
```

Reply:
done in https://github.com/infracost/infracost-azure-devops/pull/45/commits/51e7994d795d728acc2fcd1bef0007f8f0344df2